ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -216,25 +243,65 @@ function EmailIntegrationForm({
                             isDisabled={!isEditable}
                         />
                     </FormLabelGroup>
+                    <FormLabelGroup
+                        label=""
+                        fieldId="notifier.email.unauthenticated"
+                        errors={errors}
+                    >
+                        <>
+                            <div className="pf-u-display-flex pf-u-align-items-flex-start">
```

Comment:
Not a blocker to merge: See if the following example supports an equivalent popover via `FormLabelGroup` props without the need for flex alignment.

https://www.patternfly.org/v4/components/form#form-group-with-additional-label-info

Reply:
Yeah I wasn't a huge fan of having to do it this way. I originally tried adding the `isInline` prop to the FormLabelGroup component and using that but patternfly added a pretty large margin between the icon and text. So I'm not sure if theres a way to do what we want without a few utility classes. I'll dig a little more though